asciidoctor-editor-plugin/src/main/java-eclipse/de/jcup/asciidoctoreditor/preview/AsciiDoctorEditorBuildSupport.java

```diff
@@ -324,10 +325,14 @@ protected boolean isNeedingAHiddenEditorFile(File editorFileOrNull, File fileToC
 
     protected Pattern createRemoveAbsolutePathToTempFolderPattern() {
         Path tempFolder = getEditor().getWrapper().getTempFolder();
-        String absolutePathToTempFolder = tempFolder.toFile().getAbsolutePath();
```

Comment:
Interesting: 
At my linux system I used an images directory and created a test asciidoc file containing

```asciidoc
image::images folder with spaces/xasciidoctor-editor-logo.png[]
image::images folder with spaces/xdot-example.svg[]
```
Defined image content ala: 
![image](https://user-images.githubusercontent.com/11644753/68984220-26666d80-080f-11ea-98cd-48158b64b6e0.png)

When using your changes I see
![image](https://user-imaeclipse-asciidoctor-editor/asciidoctor-editor-other/testscripts/ges.githubusercontent.com/11644753/68984169-edc69400-080e-11ea-9e59-3e4d7898c171.png)


when using my origin codeScreenshot at 2019-11-16 01-18-57, I see
![image](https://user-imaeclipse-asciidoctor-editor/asciidoctor-editor-other/testscripts/ges.githubusercontent.com/11644753/68984122-c96ab780-080e-11ea-9197-f9bbe31a1f27.png)

So there seems to be still some problems.

Do you have use a windows machine? I tried out with linux.

Please do me a favor and add a similar setup like mine to 'eclipse-asciidoctor-editor/asciidoctor-editor-other/testscripts/images' and a test asciidoc file inside 
`eclipse-asciidoctor-editor/asciidoctor-editor-other/testscripts/bugfixes`
so we have a simple test case.

Reply:
The issue is not about the relative path in the adoc file. But the absolute path used when generating the preview. To reproduce you have to use spaces in the name of the parent project of the adoc file.

I'll have a look to the test setup